### ISSUE TITLE
[Bug] Weird crash with furnaces when world is open to lan

### ISSUE THREAD

**Issue by cerega000** [samolego/ClientStorage#32]
For some reason I can't open furnaces if I use "Open to LAN" option and then crash if I try opening my inventory.

How to reproduce the issue:

1. Open any world
2. Open the world to lan
3. Right click a furnace (nothing happens, furnace gui isn't showing up)
4. Press e to open inventory
5. Game crashes

Other things I've noticed
- If you hold Shift while clicking, furnace gui opens as usual, no crashes either.
- If you open any other gui (chest, barrel, crafting table, etc) after clicking a furnace, you also won't crash.
- If the mod is disabled via in-game config, furnace gui works without issues.

Crashlog: https://mclo.gs/KvKiwvO

**Comment by cerega000** [comment]
Nope. Even tried with this feature disabled. It might be better if furnaces were excluded from this mod altogether, since some items (planks, logs) can end up being wasted because they get sent into an empty furnace instead of a chest.

**Comment by samolego** [comment]
Do you have any storage presets?

**Comment by cerega000** [comment]
> Do you have any storage presets?

No
